src/containers/Sidebar.js

```diff
@@ -34,7 +34,7 @@ class Sidebar extends React.Component {
             // Created
             const creationDate = moment(level.created)
             const ago = moment.duration(moment().diff(creationDate)).asDays() || 0
-						console.log(`Level ${level.name}, created days ago: ${ago}`);
+						// console.log(`Level ${level.name}, created days ago: ${ago}`);
```

Comment:
why to keep this statement commented out? If it's not needed, it's better to remove it.